ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml

```diff
@@ -12,14 +7,22 @@ images:
 - dockerfile_path: Dockerfile
   to: special-resource-operator
 - dockerfile_literal: |
-    FROM golangci-lint
-    RUN ["apt", "update"]
-    RUN ["apt", "install", "-y", "shellcheck"]
-  from: golangci-lint
-  to: golangci-lint-shellcheck
+    FROM src
+    RUN ["yum", "update", "-y"]
+    RUN ["yum", "install", "-y", "ShellCheck"]
+    RUN ["yum", "install", "-y", "podman"]
```

Comment:
Why do we need podman?

Reply:
We needed it when I tried to build the image inside the container but I guess we don't need it if we use the current image builder infrastructure

Reply:
Removed.